client/src/components/admin/AdminPhotos.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { getbaseURL } from "@/helper/baseUrlHelper";
+import ApiService from "@/services/ApiService";
+import { computed, ref } from "vue";
+import { useRouter } from "vue-router";
+
+const router = useRouter();
+
+const year = ref(new Date().getFullYear());
+const showSpinner = ref(false);
+const archivPath = ref("");
+
+const onCreate = async () => {
+  try {
+    showSpinner.value = true;
+    const response = await ApiService.createPhotoArchivOfYear(year.value);
+    archivPath.value = response.data;
+  } catch (error) {
+    console.log(error);
+    router.push({
+      name: "NetworkError",
+    });
+  } finally {
+    showSpinner.value = false;
+  }
+};
+
+const archivDownloadUrl = computed(() => {
+  const baseURL = getbaseURL();
+  const escapedPath = archivPath.value.replaceAll("/", "%2F");
+  return baseURL + "flights/photos/download/" + escapedPath;
+});
+</script>
+
+<template>
+  <div id="adminPhotoDownloadPanel" class="py-3">
+    <button
+      class="col-2 btn btn-outline-primary btn-sm bi bi-caret-right me-3"
+      :disabled="showSpinner || archivPath.length > 0"
+      @click="onCreate"
+    >
+      Erstelle Archiv
```

Comment:
A simple download button would be nicer. Why first create the archive and then download it? User doesn't care. Just create it in the background on clicking download archive.

Reply:
Weil ich den Link brauchte für das Anchor-Tag...
Ansonsten startet der Download nicht automatisch. War mir zuviel Aufwand die Lösung dazu zu finden, für eine Sache die man eh nur einmal im Jahr braucht.